hardhat.config.js

```diff
@@ -1,9 +1,12 @@
 require("@nomiclabs/hardhat-waffle");
 require("@tenderly/hardhat-tenderly");
 
+const { task } = require("hardhat/config");
```

Comment:
why do we need this?

Reply:
Without it the linter complains that its not defined. Technically it is available globally and we could add it as a lint global but I prefer explicit imports. Makes navigating to the source code easier. Types will autocomplete as well.

Reply:
sounds good